src/components/ContentActions/index.js

```diff
@@ -14,6 +14,7 @@ const useStyles = makeStyles(({ palette, typography }) => ({
   section: {
     borderBottom: "1px solid #00000040",
     paddingBottom: typography.pxToRem(28),
+    padding: 0,
```

Comment:
@nerdlet not sure why we are adding `padding zero` here, and there is `paddingButton`

Reply:
@KhadijaMahanga Um, I think I removed this...let me confirm.

Reply:
@KhadijaMahanga Yap removed it.